File: resilience_eval/benchmarks/java/function_inlining_partial_1.java
public class function_inlining_partial_1 {
void inline_this(int[] input0){
	if(input0 != null){
		input0[0] = 1;
	}
}
void function_inlining_partial(int[] input0){
	if(input0 != null){
		inline_this(input0);
	}
}
}

File: resilience_eval/benchmarks/scala/function_inlining_partial_1.scala
class function_inlining_partial_1 {
def inline_this(input0: Array[Int]) = {
	if(input0 != null){
		input0(0) = 1;
	}
}
def function_inlining_partial(input0: Array[Int]) = {
	if(input0 != null){
		inline_this(input0);
	}
}
}
